org.gnome.Epiphany.json

```diff
@@ -82,7 +90,11 @@
                                 {
                                     "type": "git",
                                     "url": "https://github.com/sass/libsass.git",
```

Comment:
I don't think libsass/sassc would change anytime in the future, but it is fine to keep them i guess

Reply:
Agree, included for completeness